Description: Upstream changes introduced in version 1.008-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 libpar-packer-perl (1.008-1) unstable; urgency=low
 .
   * New upstream release
   * Standards-Version 3.9.1 (no changes)
   * Fix licensing for myldr/env.c (BSD license)
   * Refresh copyright information
 .
 The person named in the Author field signed this changelog entry.
Author: Jonathan Yu <jawnsy@cpan.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- libpar-packer-perl-1.008.orig/Makefile.PL
+++ libpar-packer-perl-1.008/Makefile.PL
@@ -72,7 +72,7 @@ sub par_prehook {
     $cc = can_cc unless $bork;
     my $par;
     $par = fetch_par('', '', !$cc) unless ($cc or $bork);
-    my $exe  = '';
+    my $exe  = $Config::Config{_exe};
     my $dynperl = $Config::Config{useshrplib} && ($Config::Config{useshrplib} ne 'false');
 
     if ($bork) {
@@ -115,11 +115,11 @@ sub par_prehook {
 
     makemaker_args(
         MAN1PODS		=> {
-            'script/par.pl'	=> 'blib/man1/par.pl.1p',
-            'script/pp'	        => 'blib/man1/pp.1p',
-            'script/tkpp'       => 'blib/man1/tkpp.1p',
+            'script/par.pl'	=> 'blib/man1/par.pl.1',
+            'script/pp'	        => 'blib/man1/pp.1',
+            'script/tkpp'       => 'blib/man1/tkpp.1',
           ($par or $cc) ? (
-            'script/parl.pod'   => 'blib/man1/parl.1p',
+            'script/parl.pod'   => 'blib/man1/parl.1',
           ) : (),
         },
         EXE_FILES		=> [
--- libpar-packer-perl-1.008.orig/lib/PAR/Packer.pm
+++ libpar-packer-perl-1.008/lib/PAR/Packer.pm
@@ -1238,9 +1238,9 @@ sub _par_to_exe {
 
     if ($opt->{P}) {
         # write as script
-        $parl = 'par-archive';
+        $parl = 'par.pl';
         unless ( $parl = $self->_can_run($parl, $opt->{P}) ) {
-            $self->_die("$parl not found");
+            $self->_die("par.pl not found");
         }
         $self->{parl} = $parl;
     }
